joomla/joomla-cms
# updated rating stars to cleaner look.

Pull Request for Issue # .
#### Summary of Changes

Updates rating stars to a cleaner look.
#### Testing Instructions

create an article in the front end with voting enabled.
look at stars and notice they are not clearly defined especially on dark background.
![pre](https://cloud.githubusercontent.com/assets/1850089/14593469/606493ac-04f0-11e6-8993-151638274b66.JPG)

apply patch.
notice voted and non-voted stars are much cleaner.
![post](https://cloud.githubusercontent.com/assets/1850089/14593473/65b917b0-04f0-11e6-9299-48ea264dc142.JPG)
